lib/ansible/modules/windows/win_service.py

```diff
@@ -59,6 +59,14 @@
       - restarted
     default: null
     aliases: []
+  delayed:
+    description:
+      - If True, when start_mode is C(auto) the service is flagged for a delayed start.
+    default: false
+    required: false
+    choices: [ 'True', 'False' ]
```

Comment:
Some modules define this as:
```yaml
choices:
- 'True'
- 'False'
- 'yes'
- 'no'
```
to indicate that you can also specify it differently. If there is no standard, that's up to you.